applications/leaderboard/step5/Leaderboard.csproj

```diff
@@ -2,13 +2,11 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
If you can target netcoreapp3.1 even better, althoug if it requires too much changes it's fine to leave it at netcoreapp2.1